src/classes/ERR_Handler.cls

```diff
@@ -138,62 +138,155 @@ public class ERR_Handler {
     	}
     	return errors;
     }
-        
+
```

Comment:
@randi274 I could be wrong, but these look like tab indents. I think our standard is spaces for indentations, though there are some classes that for some reason have a mix of both. For any new methods in these classes, or in methods that you're updating, the indents should be changed to spaces.

Reply:
I think this is one of the mixed classes, but everything above it seems to already be tabbed in (with some strange spacing too), and I've mostly just copied that. Should I reformat the file to use all tabs/spaces? Also, have we documented that somewhere, or is this something we should discuss at a team meeting?

Reply:
I'm pretty sure that's documented, though @davidhabib also noted this in his comments.

Reply:
@randi274 Found it - https://sites.google.com/a/salesforce.com/sfdo-products-team/artifacts-docs/technical-development-resources/cumulus-coding-conventions?pli=1. The doc is a bit old. We've discussed changes to this such as always using curly-brackets for if/for blocks even if there is only a single statement within.

Reply:
Sorry, I can't tell - did you update the indentations to use spaces?